src/views/ContinuationIn/ContinuationInAuthorization.vue

```diff
@@ -175,17 +250,31 @@ export default class ContinuationInAuthorization extends Mixins(CommonMixin, Nam
     })
   }
 
-  /** Watch all components on this page and set validity in store accordingly. */
+  /** When ULC Info is no longer required, remove existing file info. */
+  @Watch('isUlcInfoRequired')
+  private onUlcInfoRequiredChanged (val: boolean): void {
+    if (!val) {
+      this.$refs.ulcInformation.onRemoveClicked()
+    }
+  }
```

Comment:
Without this, the "affidavit" file info remains in the business info object (and we end up saving it, too).